lib/Basic/CmdLineOptions.cpp

```diff
@@ -47,19 +47,19 @@ cl::opt<bool>
 DebugValidateSolver("debug-validate-solver",
                     cl::init(false));
   
-cl::opt<int>
+cl::opt<std::string>
 MinQueryTimeToLog("min-query-time-to-log",
-                  cl::init(0),
-                  cl::value_desc("milliseconds"),
-                  cl::desc("Set time threshold (in ms) for queries logged in files. "
-                           "Only queries longer than threshold will be logged. (default=0). "
-                           "Set this param to a negative value to log timeouts only."));
+                  cl::desc("Set time threshold for queries logged in files. "
+                           "Only queries longer than threshold will be logged. (default=0s)"));
 
-cl::opt<double>
+cl::opt<bool>
+LogTimedOutQueries("log-timed-out-queries",
+                   cl::init(true),
+                   cl::desc("Log timed out queries. (default=true)."));
```

Comment:
This should be default=false, to preserve current behaviour?

Reply:
Yeah, I had it that way first. But it should happen rarely and when it happens I typically want to know why without rerunning everything. But sure, I can change that.

Reply:
I guess if it's a separate option it might make sense to make it `true` by default.